src/stratis_cli/_actions/_top.py

```diff
@@ -153,9 +172,32 @@ def add_data_devices(namespace):
             .search(managed_objects)
         )
 
+        cache = [
+            MODev(info).Devnode()
+            for (_, info) in devs(props={"Tier": BlockDevTiers.Cache}).search(
+                managed_objects
+            )
+        ]
+        already_cache = [dev for dev in namespace.blockdevs if dev in cache]
+        if already_cache != []:
+            raise StratisInUseError(already_cache, "data")
```

Comment:
Please make up an enum constant for data and cache and put it in ```_action/_constants.py```. The constant will ultimately suit better than a string.

Reply:
There is an existing enum in `_stratisd_constants.py` so I've switched over to using that.